src/MgvGovernable.sol

```diff
@@ -17,6 +17,8 @@ contract MgvGovernable is MgvRoot {
       /* Set initial gasprice and gasmax. */
       setGasprice(_gasprice);
       setGasmax(gasmax);
+      /* Without optimizer enabled it fails above 79. Set default a bit lower to be safe. */
+      setMaxRecursionDepth(75);
```

Comment:
If it's safe w/o optimizer at 79, doesn't that mean that the optimizer would increase the max recursion depth? If so, wouldn't it be safe to set it to 79 as the default?

Reply:
With the assumption that I haven't overlooked some edge case that is more expensive than the testet, then yes, I would expect so. And we can decrease it runtime if needed.
What do you think? Both you and @adhusson : Do you think it should be this hardcoded value or a deployment parameter?

Reply:
It may change over time with new EIPs, right? That could be a reason for having it a config value.
But that's probably rare, so not sure that consideration should weigh much in the decision.

My intuition is that a deployment param would be good. This will make it easy to vary depending on chain, but w/o the overhead of having it as a rarely (I expect) changed config parameter.

Reply:
Constant vs config vs deployment also applies to the 7000 constant for recursion overhead

Reply:
Resolving obsolete discussion.

Reply:
It should only change with optimizations, so we decided to set it at an expected safe 75 constant value.